src/index.js

```diff
@@ -46,10 +46,8 @@ app.use("/channels", channelRouter);
 // // app.get("/", );
 app.use("/polls", pollsRouter);
 app.use("/result/:id", resultRouter);
-// postTestMessage(
-//   process.env.SLACKBOT_TEST_CHANNEL,
-//   "Changed this to postTestMessage 123"
-// );
+
+app.use("/polls", pollsRouter);
```

Comment:
is this duplicated? what's the difference between this and line 47?

Reply:
Oopsie. Fixed.